.env.example

```
@@ -90,3 +90,12 @@ HARMONIA_OTEL_EXPORTER_OTLP_LOGS_ENDPOINT=""
 HARMONIA_OTEL_EXPORTER_OTLP_HEADERS=""
 HARMONIA_OTEL_EXPORTER_OTLP_PROTOCOL="http/protobuf"
 HARMONIA_OTEL_EXPORTER_OTLP_COMPRESSION="gzip"
+
+# -----------------------------------------------------------------------------
+# POLAR.SH BILLING
+# -----------------------------------------------------------------------------
+POLAR_ACCESS_TOKEN=""
```

dotenv-linter (4.0.0)

[warning] [QuoteCharacter] The value has quote characters (', ")

(QuoteCharacter)